components/button/styling.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Styling
+page_title: Styling
+description: Learn how to apply styles to the Blazor Button component by Telerik UI.
+slug: button-styling
+tags: telerik,blazor,button,styling,
+published: True
+position: 10
+---
+
+# Styling
+
+In addition to the [primary button](#primary-button) style and to the styles that you can apply by selecting one of the [built-in themes]({%slug general-information/themes%}), you can style the Blazor Button through its `Class` attribute. This allows you to define your own CSS rules that apply to the HTML rendering.
```

Comment:
There are a few ways to style the Button component:

* Set a [primary button](#primary-button) style
* Use another [built-in theme]({%slug general-information/themes%})
* Use the [Button `Class`](#button-class) attribute